dwave_structural_imbalance_demo/interfaces.py

```diff
@@ -133,17 +133,17 @@ def solve_structural_imbalance(self, subregion='Global', year=None):
         if self._qpu:
             try:
                 imbalance, bicoloring = dnx.structural_imbalance(G, self._embedding_composite)
-                print("Ran on the QPU using EmbeddingComposite")
+                print("Ran on the QPU using dimod's EmbeddingComposite")
```

Comment:
The embedding composite is part of `dwave-system`

Reply:
Fixed, thank you for catching that!